sys/hacl/build.rs

```diff
@@ -1,5 +1,13 @@
 use std::{env, path::Path};
 
+const LIB_NAME: &str = "hacl";
+const LIB_128_NAME: &str = "hacl_128";
+const LIB_256_NAME: &str = "hacl_256";
+#[allow(dead_code)] // FIXME: remove wrong cfg guards
```

Comment:
You mean replacing cfg guards with conditional checks in `if` statements? You may want to remove checking for `x86` arch there too?

Reply:
The cfg guards are wrong right now because they are for the host system, not the target. That's why we have dead code here.